src/app/main/component/events/services/events.service.ts

```diff
@@ -121,12 +124,12 @@ export class EventsService implements OnDestroy {
     searchTitle?: string
   ): Observable<EventResponseDto> {
     let requestParams = new HttpParams();
-    requestParams = requestParams.append('page', page.toString());
-    requestParams = requestParams.append('size', quantity.toString());
-    requestParams = requestParams.append('cities', filter.cities.toString());
-    requestParams = requestParams.append('tags', filter.tags.toString());
-    requestParams = requestParams.append('eventTime', filter.eventTime.toString());
-    requestParams = requestParams.append('statuses', filter.statuses.toString());
+    requestParams = requestParams.append('page', page.toString().toUpperCase());
+    requestParams = requestParams.append('size', quantity.toString().toUpperCase());
```

Comment:
Why page and size are to upper case? It doesn't make sense because they are numbers.